EazFixer.Code/Entrypoint.cs

```diff
@@ -0,0 +1,87 @@
+using System.IO;
+using System.Linq;
+using EazFixer.Code.Other;
+using EazFixer.Code.Processors;
+
+namespace EazFixer.Code {
```

Comment:
Please follow the original code style of the project. In particular, brackets should not be placed on the same line unless for if-, for- or using-statements.